src/lib.rs

```diff
@@ -292,41 +292,40 @@ impl Child {
 
         startup_information.cb = size_of::<STARTUPINFOW>() as u32;
 
-        let process_creation_flags = PROCESS_CREATION_FLAGS(0);
-
-        // Convert command to a wide string with a null terminator.
-        let mut command_wide = command.encode_wide().chain(Some(0)).collect::<Vec<_>>();
```

Comment:
It looks like this was 0 terminated string.

I think the 0 at the end is more correct.

Reply:
Yeah looks like there should be a NULL (0) at the end:

> The maximum length of this string is 32,767 characters, including the Unicode terminating null character. If lpApplicationName is NULL, the module name portion of lpCommandLine is limited to MAX_PATH characters.

https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw